docs/advanced-topics/message-system/serialization.md

```diff
@@ -0,0 +1,10 @@
+---
+id: serialization
+title: Serialization
+sidebar_label: Serialization  
+---
+
+Instances of Serializable Types passed into an RPC as parameters will be serialized and replicated to the remote side.
```

Comment:
why does this need it's own page? couldn't this be merged with another? like the params page?

Reply:
It's the starting page for the RFC in serialisation. if i merge it into another page then that page is carrying two items. I could add links to each of the other pages to bulk it out if needed.